requirements-test.txt

```diff
@@ -1 +1,3 @@
+--requirement requirements.txt
 -e .[test]
+pre-commit
```

Comment:
Should the `pre-commit` dependency be part of the `test` section in `setup.py`

Reply:
`pre-commit` is in the `test` section, but I wasn't sure if we wanted to leave it in `requirements-test.txt` in case something changes with the configuration in `setup.py`. I'm happy to remove it from `requirements-test.txt` if we want to leave it to the `setup.py` configuration.

Reply:
Please see 370f2a2 for that change.